.github/workflows/ci.yml

```diff
@@ -62,6 +62,9 @@ jobs:
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $RETRY $MAVEN clean verify -B --strict-checksums -V -T C1 -DskipTests -P ci -pl '!:trino-server-rpm'
+      - name: Check if all commits compile
+        run: |
+          git rebase master -x '$MAVEN compile -B --strict-checksums -V -T C1 -DskipTests -P ci -pl '\''!:trino-server-rpm'\'''
```

Comment:
Ideally the command here should match what people would use when actually performing git bisect. So probably `install -DskipTests`.